Remove jQuery dependency

santi224m opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes [#41355](https://github.com/freeCodeCamp/freeCodeCamp/issues/41355) on freeCodeCamp/freeCodeCamp repository.

<!-- Feel free to add any additional description of changes below this line -->
This replaces jQuery with a JavaScript file so that when users set the content security policies to only allow files from the same server, it won't break the page.
It also moves page styling from the html file to a css file.

gikf reviewed (commented):
I've tested this on fresh boilerplate copy and it seemed fine.

Could you change the indentations to use two spaces per level?

I'm a bit confused with code formatting of couple `<code>` tags, placing angle brackets on different line, but I'm gonna refer to other's opinion regarding that.

santi224m commented:
I changed the indentations to two spaces on the three files and I also fixed the formatting on the ```<code>``` tags. Let me know if you want me to change anything else.

gikf commented on views/index.html:
```diff
@@ -2,76 +2,60 @@
     </div>
-    <hr style='margin: 50px; margin-top: 200px'>
+    <hr class="m-50 mt-200" />
     <!-- Your web-app is https, so your scripts need to be too -->
```
```suggestion
```
After removing externally linked jquery, this is not really needed for anything.

gikf reviewed (changes requested):
Thanks for the update @santi224m.

Just one more, small suggestion and I think I'd be okay with the rest.

santi224m commented:
@gikf I removed the comment, I think it should all be good now

gikf reviewed (approved):
LGTM :+1:
